developer_manual/core/acceptance-tests.rst

```diff
@@ -141,10 +141,10 @@ Lets show an example of a feature file with scenarios:
 
     Feature: provisioning
       Background:
-        Given using api version "1"
+        Given using OCS API version "1"
 
       Scenario: Getting an not existing user
-        When user "admin" sends HTTP method "GET" to API endpoint "/cloud/users/test"
+        When user "admin" sends HTTP method "GET" to OCS API endpoint "/cloud/users/test"
```

Comment:
How about: "When user "admin" sends HTTP method "GET" to the OCS API endpoint "/cloud/users/test""